README.md

```diff
@@ -1,20 +1,24 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
-
-(b) Where in your JS file should you put it, and WHY? 
+(a) What is a click handler?
+  A click handler is a function that calls the 'click' method and then will say what should happen when a user clicks the click handler.
```

Comment:
You've got the right idea, but it's not accurate to say that "the click handler calls the 'click' method." It's actually the jQuery selector - something like `$('#submit')` - that calls the `click` method, or more popularly the `on` method with an argument of `click`. The more accurate way to say this is "a click handler is a function that determines what function will be run upon clicking the item."